src/bedtools/bedtools_intersect/config.vsh.yaml

```diff
@@ -0,0 +1,265 @@
+          that B overlap 90% of A and A _also_ overlaps 90% of B.
+      
+      - name: --either_overlap
+        alternatives: -e
+        type: boolean_true
+        description: | 
+          Require that the minimum fraction be satisfied for A OR B.
+          - In other words, if -e is used with -f 0.90 and -F 0.10 this requires
+          that either 90% of A is covered OR 10% of B is covered.
+          Without -e, both fractions would have to be satisfied.
+
+      - name: --split
+        type: boolean_true
+        description: Treat "split" BAM or BED12 entries as distinct BED intervals.
+
+      - name: --genome
+        alternatives: -g
+        type: file
+        description: | 
+          Provide a genome file to enforce consistent chromosome 
+          sort order across input files. Only applies when used 
+          with -sorted option.
+        example: genome.txt
+
+      - name: --nonamecheck
+        type: boolean_true
+        description: | 
+          For sorted data, don't throw an error if the file 
+          has different naming conventions for the same chromosome 
+          (e.g., "chr1" vs "chr01").
+
+      - name: --sorted
+        type: boolean_true
+        description: | 
+          Use the "chromsweep" algorithm for sorted (-k1,1 -k2,2n) input.
+
+      - name: --names
+        type: string
+        description: | 
+          When using multiple databases, provide an alias 
+          for each that will appear instead of a fileId when 
+          also printing the DB record.
+
+      - name: --filenames
+        type: boolean_true
+        description: When using multiple databases, show each complete filename instead of a fileId when also printing the DB record.
+
+      - name: --sortout
+        type: boolean_true
+        description: When using multiple databases, sort the output DB hits for each record.
+
+      - name: --bed
+        type: boolean_true
+        description: If using BAM input, write output as BED.
+
+      - name: --header
+        type: boolean_true
+        description: Print the header from the A file prior to results.
+
+      - name: --no_buffer_output
+        alternatives: --nobuf
+        type: boolean_true
+        description: | 
+          Disable buffered output. Using this option will cause each line
+          of output to be printed as it is generated, rather than saved
+          in a buffer. This will make printing large output files 
+          noticeably slower, but can be useful in conjunction with
+          other software tools and scripts that need to process one
+          line of bedtools output at a time.
+
+      - name: --io_buffer_size
+        alternatives: --iobuf
+        type: integer
+        description: | 
+          Specify amount of memory to use for input buffer.
+          Takes an integer argument. Optional suffixes K/M/G supported.
+          Note: currently has no effect with compressed files.       
```

Comment:
Should I remove this? Is there a meta variable for this?

Reply:
It's okay to keep this one ^^